code/pizzabot.js

```diff
@@ -1,7 +1,89 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
+
+let botSays = document.getElementById('botSays');
+botSays.innerHTML = 'What would you like to order?';
+
+// make inputbox actice (the mouse/cursor is there in beginning)
+document.getElementById("answer").focus();
+
+// used to check if valid pizza is ordered
+let orderedPizza = "";
+
+const isValidPizza = (pizza) => {
```

Comment:
There is an arrow function convention which says that if we pass to our function only one argument, we can omit parentheses `()`, and rather write :

`const isValidPizza = pizza => { ... }`

than

`const isValidPizza = (pizza) => { ... }`

Important thing to notice - this convention should be applied ONLY when we pass one argument. If we would do this with multiple arguments, it would cause `SyntaxError` to occur.

Another thing about naming conventions. Functions should be named by this pattern : `validatePizza`, `checkInput`, `showForm`, `submitInput` etc. Basically, a pair of verb and noun in this order.

`isValidPizza` is actually a convention, but for variables that store `boolean` values, not functions. Good example of when to use it would be 
```
const isPizzaValid = validatePizza('pepperoni');
```